sprint5/zadanie2.js

```diff
@@ -0,0 +1,24 @@
+/*Is the String Empty?
+Create a function that returns true if a string is empty and false otherwise.
+
+Examples
+isEmpty("") ➞ true
+
+isEmpty(" ") ➞ false
+
+isEmpty("a") ➞ false
+Notes
+A string containing only whitespaces " " does not count as empty.
+Don't forget to return the result.
+If you get stuck on a challenge, find help in the Resources tab.
+If you're really stuck, unlock solutions in the Solutions tab.*/
+
+function isEmpty(s) {
+  return s === "";
+}
+
+/*Stowrzyłam funkcję o nazwie isEmpty. Argument tej funkcji nazywa się "s".
+Funkcja ta pobiera argumenty, ktora zwraca true, jeśli
+ciąg znaków jest pusty i false. Funkcja składa się z jednej instrukcji, która
+sygnalizuje zwrot argumentu funkcji zwracanej, jezeli argumenty są równe i tego samego typu.*/
+
```

Comment:
Pani Ewo, trochę za bardzo intelektualnie :) Prostszym językiem musimy to pisywać. Po kolei:
`Stowrzyłam funkcję o nazwie isEmpty` - to jest super, tylko proszę poprawić literówkę :) 
`Argument tej funkcji nazywa się "s".` - Pięknie ! :)
`Funkcja ta pobiera argumenty, ktora zwraca true, jeśli ciąg znaków jest pusty i false` - Proszę to zdanie przerobić na: Funkcja zwraca true gdy ..., lub zwraca false gdy ...
`Funkcja składa się z jednej instrukcji, która
sygnalizuje zwrot argumentu funkcji zwracanej, jezeli argumenty są równe i tego samego typu` - to jest właśnie zbyt intelektualne :) Proszę to zdanie usunąć :)